RoadMap_SwiftUI/RoadMap_SwiftUI/ContentView.swift

```diff
@@ -8,19 +8,140 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let firstAccount = "Счёт 1"
+        static let secondAccount = "Счёт 2"
+        static let thirdAccount = "Счёт 3"
+        static let profile = "Профиль"
+        static let balanceText = "Ваш текущий баланс:"
+        static let showSettings = "Показать настройки"
+        static let replanishBalance = "Пополнить баланс"
+        static let openAnAccount = "Открыть аккаунт"
+        static let openAnAccountQuestion = "Открыть аккаунт?"
+        static let sendMoney = "Перевести"
+        static let replanishQuestionTitle = "Пополнение баланса"
+        static let sendMonetQuestion = "Вы хотите денежные средства?"
+        static let balanceReplanished = "Баланс пополнен"
+        static let applyButtonText = "Да"
+        static let okButtonText = "OK"
+        static let rejectButtonText = "Отмена"
+        static let currencyText = "₽"
+        static let sendMoneyDetails = "На вашем счёте останется"
+        static let accountCreated = "Счёт открыт!"
+        static let accountNotOpened = "Счёт ещё не открыт"
+    }
+    @State private var firstBalance = 0
+    @State private var secondBalance = 0
+    @State private var thirdBalance = 0
+    @State private var isOnToggle = false
+    @State private var isBalanceReplanishTapped = false
+    @State private var isOpenAnAccountTapped = false
+    @State private var isSendMoneyTapped = false
+    @State private var isBalanceReplanished = false
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, RoadMap!")
+            ZStack {
+                HStack {
+                    setupAccountsView()
+                        .foregroundColor(.white)
+                        .padding(.leading)
+                    Spacer()
+                }
+                RoundedRectangle(cornerRadius: 10).fill(Color.green).offset(x: isOnToggle ? 150 : 0)
+                VStack(spacing: 25) {
+                    Text("\(Constants.balanceText) \(firstBalance) \(Constants.currencyText)")
+                        .foregroundColor(.white)
+                        .font(.title)
+                        .background(.orange)
+                    setupReplanishBalance()
+                    setupOpenAnAccountButton()
+                    setupSendMoneyButton()
+                }.offset(x: isOnToggle ? 150 : 0)
```

Comment:
```suggestion
                }.offset(x: isOnToggle ? : 0)
```
убрать везде ниже магические числа